global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.1.400-preview-015204",
+    "dotnet": "3.1.400-preview-015212",
```

Comment:
Look at the build link you shared, shouldn't this be 015217?

Reply:
Yeah, once the linked build finishes. I'll update this PR once it's done (and I'll back out the test changes)